GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -1084,49 +1074,31 @@ - (void)OAuthLoginWithAddScopesFlow:(BOOL)addScopesFlow
 #endif // TARGET_OS_IOS || TARGET_OS_MACCATALYST
               completion:completion];
     } else {
+      // Mock `maybeFetchToken:` method in Sign in flow.
+      if (!(authError || modalCancel)) {
+        [[[_authState expect] andReturn:nil] lastTokenResponse];
+    #if TARGET_OS_IOS && !TARGET_OS_MACCATALYST
+        // Corresponds to EMM support
+        [[[_authState expect] andReturn:authResponse] lastAuthorizationResponse];
+    #endif // TARGET_OS_IOS && !TARGET_OS_MACCATALYST
+        [[[_authState expect] andReturn:nil] lastTokenResponse];
+        [[[_authState expect] andReturn:authResponse] lastAuthorizationResponse];
+        [[[_authState expect] andReturn:authResponse] lastAuthorizationResponse];
+      }
+      
 #if TARGET_OS_IOS || TARGET_OS_MACCATALYST
       [_signIn signInWithPresentingViewController:_presentingViewController
 #elif TARGET_OS_OSX
       [_signIn signInWithPresentingWindow:_presentingWindow
 #endif // TARGET_OS_IOS || TARGET_OS_MACCATALYST
-                                     hint:_hint
+                                     hint:nil
```

Comment:
Out of curiosity, why are you passing `nil` to `hint:` here?

Reply:
The tests about the login hint are in `GIDAuthorizationUtilTest`. Here we can simply it.